circuits/cpp/src/aztec3/circuits/kernel/public/native_public_kernel_circuit_no_previous_kernel.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "init.hpp"
+#include "common.hpp"
+
+#include <aztec3/circuits/abis/public_kernel/public_kernel_inputs_no_previous_kernel.hpp>
+#include <aztec3/circuits/abis/kernel_circuit_public_inputs.hpp>
+#include <aztec3/utils/dummy_composer.hpp>
+
+namespace aztec3::circuits::kernel::public_kernel {
+
+using aztec3::circuits::abis::KernelCircuitPublicInputs;
+using aztec3::circuits::abis::public_kernel::PublicKernelInputsNoPreviousKernel;
+using DummyComposer = aztec3::utils::DummyComposer;
+
+// TODO: decide what to return.
```

Comment:
Why do we need to decide this? Should it not return `KernelCircuitPublicInputs` as it already does?

Reply:
Yes, will delete.